data/helpers.d/psql

```diff
@@ -276,7 +276,8 @@ ynh_psql_test_if_first_run() {
 			ynh_die "postgresql shoud be 9.4 or 9.6"
 		fi
 
-		systemctl start postgresql
+		ynh_systemd_action --service_name=postgresql --action=start
```

Comment:
Is it really usefull to use this helper in this case ?

Reply:
If postgresql fails to start, the helper will give us a log.
So it seems to be useful.

Reply:
ok